client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -162,6 +169,26 @@ export const needsCustomsForm = createSelector(
 	( state, orderId, siteId = getSelectedSiteId( state ) ) => [ getForm( state, orderId, siteId ) ]
 );
 
+export const getProductValueFromOrder = createSelector(
+	( state, productId, orderId, siteId = getSelectedSiteId( state ) ) => {
+		const order = getOrder( state, orderId, siteId );
+		if ( ! order ) {
+			return 0;
```

Comment:
If the order is falsy, why does it make sense to set the product value to 0?

Right now ([see related convo on source of truth of product value](https://github.com/Automattic/wp-calypso/pull/24974/files#r200180078)), `getProductValueFromOrder` is how we get the product value if the customs item does not contain the product. If both of those methods fail, does that mean that the product value is legitimately 0, or is there a problem somewhere along the way?

Reply:
As mentioned earlier, I'll remove `getProductValueFromOrder()` entirely, since that logic will be handled by the PHP side.